root/static/scripts/edit/externalLinks.js

```diff
@@ -615,6 +616,14 @@ export class ExternalLinksEditor
           error.message = l(`This URL is not allowed 
                     for the selected link type.`);
         }
+        if (error.target ===
+          URLCleanup.ERROR_TARGETS.ENTITY) {
+          error.message = exp.l(
+            `This URL is not allowed for this type of entity:
+             {entity_type_name}.`,
```

Comment:
Maybe "This URL is not allowed for “{entity_type_name}” entities" is more straightforward? (while still being translatable).

Reply:
Applied with https://github.com/metabrainz/musicbrainz-server/pull/2280/commits/277ad5922874f54f4f9077b13d8ac195aaec2764 commit.